### ISSUE TITLE
Compare delay filtering of LST-binned data with LST-binned, delay-filtered data

### ISSUE THREAD

**Issue by jsdillon** [HERA-Team/hera_cal#486]
Is there any real advantage to delay filtering before LST-binning? It would be nice not to have to delay filter before LST-binning, especially as we more to doing more of the pipeline on-site as part of RTP.

**Comment by jsdillon** [comment]
@nkern can you take this one?

**Comment by nkern** [comment]
I agree: I think we had a discussion on this a while back and I argued for the latter. We went with the former b/c of delay-XRFI, so we can perhaps eliminate delay filtering from the analysis pipeline.

**Comment by jsdillon** [comment]
I'd love to see some plots showing it's not necessary.

**Comment by jsdillon** [comment]
I think this is something we need to know for architecting the new RTP. Hopefully we can answer this question with IDR2.2. @nkern do you still want to take this on, or should we find someone else to do it?

**Comment by jsdillon** [comment]
Discussed 9/4:

@nkern Argued that this isn't the lowest hanging fruit right now and wanted to punt on this question, which means the delay filter from the pipeline.

@jsdillon Suggested that it would be useful to verify empirically that we can always filter after LST-binning. @jsdillon to take a look at real-space data. Perhaps @saurabh-astro might be interested in taking up this problem using hera-pspec?

@jaguirre Argued that it's not clear if the delay filter is ever useful, especially for power spectrum estimation

@anguta Argued that it might be useful for assessing systematics, but it would be hard to validate for power spectrum estimation.

**Comment by jsdillon** [comment]
I did a little bit of digging into this. I looked at delay filtering `zen.grp1.of1.LST.1.31552.HH.OCRSL.uvh5` and compared it to `zen.grp1.of1.LST.1.31552.HH.OCRSDL.uvh5`. I'd say the results are comparable, though there are areas where one is larger and areas where the other is larger. For shorter baselines, the residual of bin-then-filter is larger near the band edges and overall a bit higher as a mean, but lower as a median. If we flag all visibilities with nsamples < .5 max(nsamples) before filtering, then pretty consistently we get lower means and median residuals for LST-binning then filtering. For future analysis, if we want to delay filter after LST-binning, we should look into weighting it by nsamples. We also may still want to use the delay filtered data to hunt for residual RFI after LST-binning, but that's a separate issue. 

I feel pretty good saying that we can delay filter after LST binning. There's no strong evidence for why we can't and it might even be a bit better. I'm closing this unless @nkern objects.
